full/src/test/java/apoc/trigger/TriggerClusterTest.java

```diff
@@ -47,7 +45,7 @@ public void before() {
         cluster.getSession().run("MATCH (n) DETACH DELETE n");
     }
 
-
+    @Ignore
```

Comment:
Yes in this case we are ignoring all of these tests because the cluster can't start. The failure it gives for TriggerClusterTest is that the routing table of the database cluster is out of date. I think this will be fixable but I'd like to progress by stages, but removing these @Ignore is something I'd like us to do.